client/auth/tokens.go

```diff
@@ -0,0 +1,40 @@
+package auth
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/SKF/go-utility/v2/auth/secretsmanagerauth"
+)
+
+type IdentityToken string
```

Comment:
Do we want to call it token or accesstoken? instead. or authorization.

The fact that we are using identitytoken for the authorization header is an enlight specific detail.